terraform/slack-trigger/variables.tf

```diff
@@ -3,8 +3,13 @@ variable "name_prefix" {
   type        = string
 }
 
+variable "slack_signing_secret_name" {
+  description = "Name of the parameter containing the slack signing secret"
+  type        = string
+}
+
 variable "slack_signing_secret_arn" {
```

Comment:
is the `slack_signing_secret_arn` needed for the slack-trigger or can this block be removed?

Reply:
ah yes it's needed to build the IAM role to access it https://github.com/nick-solly/knimer/pull/1/files/1eea4013bc9b0f7e44c82f664b74632dde3a9686#diff-27ba68ec068c9a6ec6507d449953b624601d4fbadf7cc1f7972d81cd79f48205R41